### PR TITLE
Add Turkish translation

### PR DESCRIPTION
**Pull request by safak45x**
Please add Turkish translation
Send error message will see error

### CODE REVIEW

File: po/com.github.cassidyjames.ephemeral.pot [hunk 1]
```diff
@@ -269,59 +279,63 @@ msgid ""
...
 #, c-format
 msgid "Go to \"%s\""
-msgstr ""
+msgstr "\"%s\ Adresine git"
 
 #: src/Widgets/UrlEntry.vala:185
 #, c-format
 msgid "Search for \"%s\""
-msgstr ""
+msgstr "\"%s\ Şunu Ara"
```
Comment (cassidyjames) [inline po/com.github.cassidyjames.ephemeral.pot]: This is a quoted string, so it should probably be:

```suggestion
msgstr "\"%s\" Şunu Ara"
```

Comment (safak45x) [reply po/com.github.cassidyjames.ephemeral.pot]: I'm fixing now

**Comment by cassidyjames** [comment]
Thank you!

**Comment by safak45x** [comment]
> Teşekkür ederim!
bugs fixed
can I ask a question

**Comment by cassidyjames** [comment]
>can I ask a question

Yes, what's your question?

**Comment by safak45x** [comment]
> Yes, what's your question?
Translation for application 4 or 5 
AppCenter 1 week but didn't update the application
Don't put a developer myself
I have to wait or more
Sorry for kind of bad English expression I

**Comment by safak45x** [comment]
> > can I ask a question
> 
> Yes, what's your question?

5 application to translate into Turkish
Translation appcenter application not for 1 Week
I have to wait more 
with the developer or oynuyacakmi
The Turkish narrative is a little bad I'm sorry I

**Comment by cassidyjames** [comment]
@safak45x it's up to each developer to create a new release of their app once a translation is merged. After the release is made, it might take some time for the update to be reviewed by elementary. But those translations should make it into an update eventually.

**Comment by safak45x** [comment]
> @safak45x it's up to each developer to create a new release of their app once a translation is merged. After the release is made, it might take some time for the update to be reviewed by elementary. But those translations should make it into an update eventually.

okey thank you for answering my question

**Pull request merged**